### PR TITLE
Improve documentation for error responses

### PR DESCRIPTION
**Pull request by lkmorlan**
The documentation for error responses does not make clear that an array of error messages will be returned. This pull request describes the format of the response and provides an example.

### CODE REVIEW

File: src/en/send.md [hunk 1]
```diff
@@ -219,7 +219,23 @@ If the request to the client is successful, the client returns a `dict`:
 
 ### Error codes
 
-If the request is not successful, the response body is `json`. Refer to the table below for details.
+If the request is not successful, the response body is `json` with the key `status_code` containing the integer status code and the key `errors` containing an array of objects each of which has keys `error` and `message`. Refer to the table below for status codes and example error objects. Example error response:
```
Comment (jzbahrai) [inline src/en/send.md:222]:
```suggestion
If the request is not successful, the response body is `json` with the key `status_code` containing the integer status code and the key `errors` containing an array of objects each of which has keys `error` and `message`. Refer to the table below for status codes and example error objects. Example error response:
```

**Comment by jzbahrai** [comment]
@lkmorlan thank you so much for submitting this! We welcome contributors and are very happy to receive a PR!
 We just want to change the wording a little. Can you accept my suggestion? We will also add the french copy for this.

**Comment by jzbahrai** [comment]
@jimleroyer when you get a moment if you can add the french translation for this.

**Comment by jimleroyer** [comment]
I had to create a new PR #130 as the CI pipeline won't run properly without appropriate authorizations. Thank you @lkmorlan for the contribution!

**Pull request closed without merging**